DEPS

```diff
@@ -21,6 +21,10 @@ deps = {
   "third_party/ethash/src": "https://github.com/chfast/ethash.git@e4a15c3d76dc09392c7efd3e30d84ee3b871e9ce",
   "third_party/bitcoin-core/src": "https://github.com/bitcoin/bitcoin.git@8105bce5b384c72cf08b25b7c5343622754e7337", # v25.0
   "third_party/argon2/src": "https://github.com/P-H-C/phc-winner-argon2.git@62358ba2123abd17fccf2a108a301d4b52c01a7c",
+  "third_party/libdmg-hfsplus": {
```

Comment:
is this the only way to get this python module? I think there's a better method than adding them to deps cc @goodov

Reply:
This is not a Python module. It's a tool that's written in C.